book/4-clustering.tex

```diff
@@ -92,6 +92,42 @@ \subsection{Homogeneity Completeness V Measure}
 \section{Silhouette Score}
 \subsection{Silhouette Score}
 
+The Silhouette Coefficient is a measure of how well samples are clustered with samples that are similar to themselves.
+
+The Silhouette Coefficient s for a given sample is given as 
+
+\begin{center}
+\tikz{
+\node[inner sep=2pt, font=\Large] (a) {
+{
+$\displaystyle
+s = \frac{{\color{cyan}b} - {\color{nmlpurple}a}}{\max({\color{cyan}b}, {\color{nmlpurple}a})}
+$
+}
+};
+}
+\end{center}
+
+\textbf{a} : The mean distance between a sample and all other points in the same cluster (mean intra-cluster distance)
+
+\textbf{b} : The mean distance between a sample and all other points in the nearest cluster (mean nearest-cluster distance)
+
+Silhouette Score is the average of all the Silhouette Coefficients of each sample in the dataset.
```

Comment:
Ideally, after the formula, we only have one paragraph. Try to condense this ideas into a single paragraph.